test/run-example-tests

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+#
+# Runs the ringpop-admin tests.
+#
+# This script launches a ringpop cluster using tick-cluster and then runs
+# a series of tests against the ringpop-admin command output.
+#
+set -euo pipefail
+
+
+declare cram_opts="-v --shell=/bin/bash"
```

Comment:
I don't like the formatting of the README.md files now. Due to only two spaces they are hard to read on github. if we add `--indent=4` to the cram options we can indent the code blocks by 4 spaces which will put a box around them in markdown making it a bit easier to read IMHO.